logging.properties

```diff
@@ -0,0 +1,4 @@
+handlers= java.util.logging.ConsoleHandler
```

Comment:
Why does this file have to be added?

Reply:
MiniSSL.java utilizes some logging methods via JUL (Java Utility Logging).
I found it useful for testing to include this properties file so I can turn on more finer log statements as I developed this change.

I describe it in the _JRUBY.md_. It would not be distributed with Puma and is merely a development productivity file.

Reply:
are you sure `LOGGER.info("Using JDK SSL engine");` won't get distributed?
instead of these you might consider not using JUL and simply using (J)Ruby's `debug/verbose` flag.
or a system property `-Dpuma.ssl.debug=true` to turn on simple (std-out) logging.